eclair-core/src/test/scala/fr/acinq/eclair/channel/states/g/NegotiatingStateSpec.scala

```diff
@@ -52,7 +52,14 @@ class NegotiatingStateSpec extends TestkitBaseClass with StateTestsHelperMethods
       reachNormal(setup)
       val sender = TestProbe()
       // alice initiates a closing
-      if (test.tags.contains("fee2")) Globals.feeratesPerKw.set(FeeratesPerKw.single(4319)) else Globals.feeratesPerKw.set(FeeratesPerKw.single(10000))
+      if (test.tags.contains("fee2")) {
+        feeConfOfChannel(alice).feeEstimator.asInstanceOf[TestFeeEstimator].setFeerate(FeeratesPerKw.single(4319))
```

Comment:
it would be better if we could write `setFeeRate(alice, FeeratesPerKw.single(4319))` and not have to cast `feeEstimator` here

Reply:
9da4996 uses pimp pattern to make `TestFSMRef` (both for Channel and Peer) return an instance of `TestFeeEstimator` and `FeeTargets`